openshift/olm/knative-serving.catalogsource.yaml

```diff
@@ -555,11 +555,201 @@ data:
+          mediatype: image/png
+        install:
+          spec:
+            clusterPermissions:
+            - rules:
+              - apiGroups:
+                - '*'
+                resources:
+                - '*'
+                verbs:
+                - '*'
+              serviceAccountName: knative-serving-operator
+            deployments:
+            - name: knative-serving-operator
+              spec:
+                replicas: 1
+                selector:
+                  matchLabels:
+                    name: knative-serving-operator
+                strategy: {}
+                template:
+                  metadata:
+                    labels:
+                      name: knative-serving-operator
+                  spec:
+                    containers:
+                    - command:
+                      - knative-serving-operator
+                      env:
+                      - name: WATCH_NAMESPACE
+                        valueFrom:
+                          fieldRef:
+                            fieldPath: metadata.annotations['olm.targetNamespaces']
+                      - name: POD_NAME
+                        valueFrom:
+                          fieldRef:
+                            fieldPath: metadata.name
+                      - name: OPERATOR_NAME
+                        value: knative-serving-operator
+                      image: quay.io/openshift-knative/knative-serving-operator:v0.7.1
+                      args:
+                        - --filename=https://raw.githubusercontent.com/openshift/knative-serving/release-v0.6.0/openshift/release/knative-serving-v0.6.0.yaml
```

Comment:
Might be chicken/egg because 0.7.1 is not yet cut, so nothing to reference. This is mostly an update for release-next so i think that's fine for now?

Reply:
I'd probably delete it then.

Reply:
@jcrossley3  we can't delete that line because in 
e2e-tests-openshift.sh file we are doing sed operation based on --filename

https://github.com/openshift/knative-serving/blob/master/openshift/e2e-tests-openshift.sh#L86
https://github.com/openshift/knative-serving/blob/master/openshift/e2e-tests-openshift.sh#L87